src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Network Analysis/FindServiceArea/FindServiceArea.xaml.cs

```diff
@@ -0,0 +1,264 @@
+
+        // Used to see whether the user is placing barriers or facilities
+        private Boolean _barrierMode = false;
+        private Boolean _facilityMode = false;
+
+        // Uri for service area
+        private Uri _sanDiegoServiceAreaUri = new Uri("https://sampleserver6.arcgisonline.com/arcgis/rest/services/NetworkAnalysis/SanDiego/NAServer/ServiceArea");
+
+        // Picture for facility symbol
+        private Uri _pictureMarkerUri = new Uri("http://static.arcgis.com/images/Symbols/SafetyHealth/Hospital.png");
+
+        public FindServiceArea()
+        {
+            InitializeComponent();
+            // Create the map, graphics overlay, and the 'from' and 'to' locations for the route.
+            Initialize();
+        }
+        
+        private void Initialize()
+        {
+            // Center the map on San Diego.
+            Map map = new Map(Basemap.CreateStreets());
+            MyMapView.Map = map;
+            MyMapView.SetViewpointCenterAsync(32.73, -117.14, 30000);
+
+            // Create graphics overlays for all of the elements of the map.
+            _serviceAreasOverlay = new GraphicsOverlay();
+            _facilityOverlay = new GraphicsOverlay();
+            _barrierOverlay = new GraphicsOverlay();
+            MyMapView.GraphicsOverlays.Add(_serviceAreasOverlay);
+            MyMapView.GraphicsOverlays.Add(_facilityOverlay);
+            MyMapView.GraphicsOverlays.Add(_barrierOverlay);
+
+            // Link the action of tapping on the map with the MyMapView_GeoViewTapped method.
+            MyMapView.GeoViewTapped += MyMapView_GeoViewTapped;
+
+            // Create a polyline builder for user constructed barriers.
+            _barrierBuilder = new PolylineBuilder(_spatialReference);
+
+            // Create a list of serviceAreaFacilities.
+            _serviceAreaFacilities = new List<ServiceAreaFacility>();
+
+            // Symbology for a service area.
+            _outline = new SimpleLineSymbol(SimpleLineSymbolStyle.Solid, System.Drawing.Color.Red, 3.0f);
+
+            // Create a list of Fill Symbols for the polygons.
+            _fillSymbols = new List<SimpleFillSymbol>();
+            _fillSymbols.Add(new SimpleFillSymbol(SimpleFillSymbolStyle.Solid, System.Drawing.Color.FromArgb(70, 255, 0, 0), _outline));
+            _fillSymbols.Add( new SimpleFillSymbol(SimpleFillSymbolStyle.Solid, System.Drawing.Color.FromArgb(70, 255, 165, 0), _outline));
+
+            // Symbology for a facility
+            _facilitySymbol = new PictureMarkerSymbol(_pictureMarkerUri);
+            _facilitySymbol.Height = 30;
+            _facilitySymbol.Width = 30;
+
+        }
+        private async Task CreateServiceArea()
+        {
+            // Create the service area task and paramaters based on the Uri.
+            _serviceAreaTask = await ServiceAreaTask.CreateAsync(_sanDiegoServiceAreaUri);
+            _serviceAreaParameters = await _serviceAreaTask.CreateDefaultParametersAsync();
```

Comment:
I think `CreateDefaultParametersAsync` might need some comments/description. It's an important step in building the params and saves a lot of headaches versus creating everything from the ground up.